lib/ctcs2_to_junit.pm

```diff
@@ -99,15 +99,15 @@ sub generateXML {
             name      => $test,
             status    => $case_status,
             time      => $test_results{$test}->{time});
-        if ($case_status eq 'failure' || $case_status eq 'skipped') {
+        if (!get_var('QA_TESTSUITE') && ($case_status eq 'failure' || $case_status eq 'skipped')) {
             (my $test_path = $test) =~ s/-/\//;
             $test_path = '/opt/log/' . $test_path;
-            my $test_out_content = script_output("cat $test_path\n", 600);
+            my $test_out_content = script_output("cat $test_path| sed \"s/'//g\"", 600);
```

Comment:
a thing which people usually forget when using `script_out` that function will fail the whole test case if regex will not match something. Just confirm that such behavior would be expected here